src/main/java/org/folio/search/utils/LogUtils.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.search.utils;
+
+import java.util.Collection;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class LogUtils {
```

Comment:
I'd say it's not necessary. We usually log only size of list. And even list with size = 2 may be too big (if large objects stored) to put in log.
It should be decided individually when logging.